script/BaseScript.s.sol

```diff
@@ -54,30 +52,22 @@ struct BaseInput {
 struct Deployment {
     ChainDeployment[] chains;
     address deployerAddress;
-    string erc20Name;
```

Comment:
remove erc20Name and erc20Symbol from struct Deployment, since they were not being used anywhere.

Reply:
ok cool